### ISSUE TITLE
Possible redundant parameter `mlp_dim`  in `class Transformer` definition?

### ISSUE THREAD

**Issue by gurdeepmaurya** [lucidrains/vit-pytorch#344]
Phil, my man,

Thanks for writing such a beautiful repo. 

I think `mlp_dim` in the `class Transformer`  definition may be a redundant parameter, 'cause there doesn't seem to be any situation where `mlp_dim` is not equal to `dim`.

https://github.com/lucidrains/vit-pytorch/blob/297e7d00a20628ad075470362c215095dbf5c7bd/vit_pytorch/simple_vit.py#L72

If there's a use case for `dim!=mlp_dim`, then the code implementing residual connection may not work.

https://github.com/lucidrains/vit-pytorch/blob/297e7d00a20628ad075470362c215095dbf5c7bd/vit_pytorch/simple_vit.py#L77

Let me know your thoughts

< Dap Up >

**Comment by gurdeepmaurya** [comment]
Just saw the definition of `class FeedForward(nn.Module)`  , and I see that it's not redundant. Closing this issue.
